crosswalks/usgs-modelcatalog.csv

```diff
@@ -0,0 +1,74 @@
+Property,usgs-modelcatalog
+codeRepository,software.usgs_software_gitlab_release_urls / software.external_software_repository_urls
+programmingLanguage,programming_language
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,item_type
+applicationSubCategory,type_keywords
+downloadUrl,software.download_url
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,person.name when person_type=Author
+citation,publications.model_citations
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,last_update
+datePublished,release_date
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,science_keywords / other_keywords
+license,license
+producer,organization / external_organization_name
+provider,
+publisher,
+sponsor,
+version,version
+isAccessibleForFree,
+isPartOf,related_modelcatalog_assets
+hasPart,related_modelcatalog_assets
+position,
+description,description
+identifier,identifier.name / identifier.id / identifier.url / identifier.type
+name,name
+sameAs,
+url,other_links.usgs_website / other_links.weblinks
+relatedLink,other_links.weblinks
```

Comment:
```suggestion
url,other_links.usgs_website
relatedLink,other_links.weblinks
```

no need to duplicate. `url` should be the homepage of the project or something similar, so I'm guessing `relatedLink` is a better match for "other links" (especially with the definition saying "Examples include community forums for the model.")

Reply:
I agree mostly, however some complexity is added because some of the models have an official usgs website, while other models do not have an official page on usgs.gov/, so their main page will be in weblinks instead of usgs_website. That should be fine, since all links will be represented, although it bothers me that it is not consistent, due to choices we made for our system.